src/constants/default.ts

```diff
@@ -1,6 +1,8 @@
 export const ETH_STANDARD_PATH = "m/44'/60'/0'/0";
 export const ENTROPY_KEY = "ENTROPY";
-export const MNEMONIC_KEY = "MNEMONIC";
+// export const MNEMONIC_KEY = process.env.MNEMONIC ?? "MNEMONIC";
```

Comment:
any ideas about how to get this MNEMONIC from environment variables while we're here in the walletconnect test wallet service?